Homeworks/Homework SQL/SQLQuery-Homework.sql

```diff
@@ -0,0 +1,339 @@
+INSERT INTO Libraries
+VALUES ('Library 5','Strada Stefan Cel Mare','Iasi');
+INSERT INTO Libraries
+VALUES ('Library 6','Bv. Socola','Iasi');
+
+-- Forgot to change library number
+DELETE FROM Libraries
+WHERE Id=6;
+
+SELECT * 
+FROM Libraries;
+
+INSERT INTO IsOnHoliday
+VALUES ('True');
+INSERT INTO IsOnHoliday
+VALUES ('False');
+
+SELECT *
+FROM IsOnHoliday;
+
+INSERT INTO Librarians
+VALUES ('Gustav','2022-01-01',1,1);
+INSERT INTO Librarians
+VALUES ('Robert','2010-01-01',7,2);
+INSERT INTO Librarians
+VALUES ('Giulia','2012-01-01',8,1);
+INSERT INTO Librarians
+VALUES ('Maxi','2019-01-01',9,2);
+INSERT INTO Librarians
+VALUES ('Vlad','2013-01-01',10,2);
+INSERT INTO Librarians
+VALUES ('Dana','2023-01-01',1,2);
+INSERT INTO Librarians
+VALUES ('Mihai','2007-01-01',11,1);
+INSERT INTO Librarians
+VALUES ('Ionut','2019-01-01',8,2);
+
+INSERT INTO Sales
+VALUES(13,1,1);
+INSERT INTO Sales
+VALUES(33,1,1);
+INSERT INTO Sales
+VALUES(312,7,2);
+INSERT INTO Sales
+VALUES(112,8,3);
+INSERT INTO Sales
+VALUES(53,9,4);
+INSERT INTO Sales
+VALUES(441,10,5);
+INSERT INTO Sales
+VALUES(444,11,6);
+INSERT INTO Sales
+VALUES(44,8,7);
+INSERT INTO Sales
+VALUES(124,9,8);
+INSERT INTO Sales
+VALUES(22,1,9);
+INSERT INTO Sales
+VALUES(44,7,10);
+INSERT INTO Sales
+VALUES(121,10,11);
+
+INSERT INTO Books_Sales
+VALUES (1,1);
+
+INSERT INTO Books_Sales
+VALUES (2,2);
+INSERT INTO Books_Sales
+VALUES (3,3);
+INSERT INTO Books_Sales
+VALUES (4,4);
+INSERT INTO Books_Sales
+VALUES (5,5);
+INSERT INTO Books_Sales
+VALUES (5,6);
+INSERT INTO Books_Sales
+VALUES (4,7);
+INSERT INTO Books_Sales
+VALUES (3,8);
+INSERT INTO Books_Sales
+VALUES (2,9);
+INSERT INTO Books_Sales
+VALUES (1,10);
+INSERT INTO Books_Sales
+VALUES (2,11);
+INSERT INTO Books_Sales
+VALUES (4,12);
+
+INSERT INTO Books_Libraries
+VALUES(1,1);
+INSERT INTO Books_Libraries
+VALUES(2,7);
+INSERT INTO Books_Libraries
+VALUES(3,8);
+INSERT INTO Books_Libraries
+VALUES(4,9);
+INSERT INTO Books_Libraries
+VALUES(5,10);
+INSERT INTO Books_Libraries
+VALUES(6,11);
+INSERT INTO Books_Libraries
+VALUES(7,1);
+INSERT INTO Books_Libraries
+VALUES(8,7);
+INSERT INTO Books_Libraries
+VALUES(9,8);
+INSERT INTO Books_Libraries
+VALUES(2,9);
+INSERT INTO Books_Libraries
+VALUES(3,10);
+INSERT INTO Books_Libraries
+VALUES(4,11);
+INSERT INTO Books_Libraries
+VALUES(5,1);
+
+-- See all Tables
+SELECT *
+FROM Author;
+SELECT *
+FROM Books;
+SELECT *
+FROM Books_Libraries;
+SELECT *
+FROM Books_Sales;
+SELECT *
+FROM Librarians;
+SELECT *
+FROM Libraries;
+SELECT *
+FROM Sales;
+
+--2. Display all the librarians together with the library name they work in ordered by librarian name and library name descending.
+
+SELECT l.Name,l.HireDate,l.IsOnHolidayId as HolidayStatus, i.Name
+FROM Librarians l
+INNER JOIN Libraries i ON i.Id = l.LibraryId
+ORDER BY l.Name ASC,i.Name DESC;
+
+--3. Update all the street names to start with "STR.".
+
+UPDATE LIbraries
+SET
+    StreetName = 'STR.'+StreetName
+WHERE Id=11;
```

Comment:
All the street names should be updated not only one. Also, there is no library with Id 11 in my database as I only have 6 insert commands above (this means ids from 1 to 6).